justfile

```diff
@@ -24,3 +28,7 @@ test *args="//...":
 
 test-asan *args="//...":
   just test {{args}} --config=asan
+
+build-test *args="//...":
+  just build {{args}}
```

Comment:
Nit: I'd go with a combined build-test step (i.e. deleting this line) which is slightly faster and will often show failing test results (if any) before having to build everything.